### PR TITLE
chore: move the view APIs to their own directory

### PR DESCRIPTION
**Pull request by MarshallOfSound**
The `api` folder was getting quite big and these are all in one feature, makes sense to isolate them in a directory

Notes: no-notes

### CODE REVIEW

**Comment by codebytere** [comment]
I'd probably also advocate for moving the corresponding cpp files into their own directory

**Comment by nitsakh** [comment]
Yeah, agree with @codebytere. Would be nice to move the cpp files also.

**Comment by codebytere** [comment]
bump @MarshallOfSound do you gave time to move the cpp files for this? i can also push that to this PR if you'd want?

**Comment by MarshallOfSound** [comment]
@codebytere If you get time today your time, go for it, if not I should have some time tomorrow to circle back to this 👍

File: atom/browser/api/views/atom_api_box_layout.cc [hunk 1]
```diff
@@ -2,11 +2,11 @@
 // Use of this source code is governed by the MIT license that can be
 // found in the LICENSE file.
 
-#include "atom/browser/api/atom_api_box_layout.h"
+#include "atom/browser/api/views/atom_api_box_layout.h"
 
 #include <string>
 
-#include "atom/browser/api/atom_api_view.h"
+#include "atom/browser/views/api/atom_api_view.h"
```
Comment (deepak1556) [inline atom/browser/api/views/atom_api_box_layout.cc]: `"atom/browser/views/api/atom_api_view.h"` -> `"atom/browser/api/atom_api_view.h"`

**Pull request merged**